api/server/components.ts

```diff
@@ -0,0 +1,20 @@
+export {CacheClient} from '../../src/server/components/cache-client';
+export {
+    getLandingLayout,
+    SDK,
+    Utils,
+    getChartkitLayoutSettings,
+    getPlatform,
+} from '../../src/server/components';
+export {getAppLayoutSettings} from '../../src/server/components/app-layout/app-layout-settings';
+export * as resolveEntryByLink from '../../src/server/components/resolve-entry-by-link';
+export * as metrikaDataFormatter from '../../src/server/components/metrika-data-formatter';
```

Comment:
Would be better not to use the `*` exports

Reply:
Yep, I removed them